### DISCUSSION TITLE
Practica Html-Taqueria

### DISCUSSION THREAD

**Discussion by adrianBurelos** [Launch-X-Latam/MisionFrontEnd#369] [Show and tell]
Buenas Tardes
Compañeros subo la practica de la taqueria 

link repositorio: https://github.com/adrianBurelos/LaunchX-Taqueria
link pagina: https://adrianburelos.github.io/LaunchX-Taqueria/

**Comment by Jramo5** [comment]
Esta guapa, me gusto el diseño
